explorer/interpreter/action_stack.cpp

```diff
@@ -262,4 +314,35 @@ void ActionStack::SetResult(Nonnull<const Value*> result) {
   }
 }
 
+void ActionStack::DestroyAllScopes(
+    std::list<std::unique_ptr<Action>>&& actions) {
```

Comment:
```suggestion
    std::list<std::unique_ptr<Action>> actions) {
```

See the [style guide](https://google.github.io/styleguide/cppguide.html#Rvalue_references) for more details about why. This also applies to `DestroyScopes`.

Reply:
Done